src/events.js

```diff
@@ -46,7 +58,7 @@ module.exports = function(ctx) {
   };
 
   events.drawChanged = function(event) {
-    featuresChanged = event.features;
+    event.features.forEach(f => changedFeatureIds.add(f.id));
```

Comment:
As noted in https://github.com/mapbox/mapbox-gl-draw/issues/367, I think the "changed" vs "modified" situation is very confusing ... But certainly a more precise variable might help here, like `changedSinceLastModified` (or better word than "modified" I'd hope?)?